file_paths.md

```diff
@@ -339,6 +340,39 @@ The package will install a service named `puppetmaster`, create a
         puppetmaster *                    # writeable by puppetmaster
             puppetmaster.pid
 
+# pxp-agent
+
+This package executes Puppet and replies to PXP requests.
+
+## *nix
+
+    /opt/puppetlabs/pxp-agent *
+        spool                             # :spooldir
```

Comment:
Are `:spooldir`, `:modulesdir`, etc puppet settings? or pxp-agent settings? We originally listed puppet settings in this document so it was clear how to obtain those values in an acceptance test using `puppet agent --configprint vardir` for example. If those are pxp-agent settings I'm not sure how to disambiguate so people don't think they are puppet settings.